### ISSUE TITLE
Template tag ignored

### ISSUE THREAD

**Issue by ehrencrona** [sveltejs/svelte-preprocess#231]
**Describe the bug**
I tried to get Pug support running, but it doesn't seem to process the `template` tag at all. HTML output is empty unless I include a `main`. tag.

**To Reproduce**

I tried setting up a new project according to the instructions in "Getting started":

```
npx degit sveltejs/template my-svelte-app
cd my-svelte-app
yarn add --dev svelte-preprocess
yarn add pug
```

I then add
```
preprocess: sveltePreprocess(),
```
to `rollup.config.js`,

and replace `<main>` in `App.svelte` with

```
<template lang="pug">
  h1 {name}
</template>
```

When running `yarn run dev` I get an empty page. Introducing a syntax error in the Pug template does not yield an error message so I assume it's not processed at all.

Repro at https://github.com/ehrencrona/svelte-pug-template-ignored

I assume I've done some stupid mistake but I can't figure out what I and think I've followed the getting started instructions exactly.

**Expected behavior**
I'd expect it render `<h1>World</h1>`

**Information about your project:**

svelte-preprocess@4.1.1
svelte@3.24.1
pug@3.0.0

(on a Mac)

**Comment by ehrencrona** [comment]
Ok, I've figured it out. It must be

```
<template lang="pug">
h1 {name}
</template>
```

rather than

```
<template lang="pug">
  h1 {name}
</template>
```

The example in "getting started" is wrong then. This feels a bit confusing, there ought to be at least a warning if you start the template with spaces.

**Comment by kaisermann** [comment]
Hey @ehrencrona 👋  

The preprocessor should be removing the indentation for you. I have an idea of what's happening and will try to fix it later today!

**Comment by ehrencrona** [comment]
I was just looking at it. It seems that the mixins that are prepended mean that if the template itself is indented, it's considered part of the last mixin.

My suggestion would have been to de-indent the code in the preprocessor (i.e. find the longest common sequence of indent characters at the beginning of any line and strip it out). Should I make a PR with that?

**Comment by kaisermann** [comment]
@ehrencrona We already have a method to de-indent it [here](https://github.com/sveltejs/svelte-preprocess/blob/master/src/modules/prepareContent.ts#L14). The default preprocessor should be setting the `stripIndent` to `true`, but the [line responsible](https://github.com/sveltejs/svelte-preprocess/blob/master/src/autoProcess.ts#L137) for that is never reached because `name !== alias` is always `false` for `pug`.

**Comment by ehrencrona** [comment]
Oh, I see. I'll leave it to you then :P Thanks!

**Comment by kaisermann** [comment]
Fixed in `4.1.2` 🎉
